charts/hub-net-controller-manager/README.md

```diff
@@ -39,5 +39,40 @@ helm upgrade hub-net-controller-manager ./charts/hub-net-controller-manager/
 | podAnnotations | Pod Annotations | `{}` |
 | affinity | The node affinity to use for pod scheduling | `{}` |
 | tolerations | The toleration to use for pod scheduling | `[]` |
+| config.azureCloudConfig | The Azure cloud provider configuration | **required if AzureTrafficManager feature is enabled** |
+
+## Override Azure cloud config
+
+**If AzureTrafficManager feature is enabled, then an Azure cloud configuration is required.** Azure cloud configuration provides resource metadata and credentials for `fleet-hub-net-controller-manager` and `fleet-member-net-controller-manager` to manipulate Azure resources. It's embedded into a Kubernetes secret and mounted to the pods. The values can be modified under `config.azureCloudConfig` section in values.yaml or can be provided as a separate file.
```

Comment:
can you add a param to enable azure traffic manager feature?

so that we only install the azure cloud config when the feature is on.